docs/yul.rst

```diff
@@ -9,12 +9,9 @@ Yul
 Yul (previously also called JULIA or IULIA) is an intermediate language that can be
 compiled to bytecode for different backends.
 
-Support for EVM 1.0, EVM 1.5 and Ewasm is planned, and it is designed to
-be a usable common denominator of all three
-platforms. It can already be used in stand-alone mode and
-for "inline assembly" inside Solidity
-and there is an experimental implementation of the Solidity compiler
-that uses Yul as an intermediate language. Yul is a good target for
+It can be used in stand-alone mode and
+for "inline assembly" inside Solidity.
+The IR-based code generator (“new codegen” or “IR-based codegen”) uses Yul as an intermediate language. Yul is a good target for
```

Comment:
I'd wrap at the end of sentence and use normal quotes (the fancy ones may technically be more correct, but I don't think we're using them anywhere else).
```suggestion
It can be used in stand-alone mode and for "inline assembly" inside Solidity.
The IR-based code generator ("new codegen" or "IR-based codegen") uses Yul as an intermediate language.
Yul is a good target for
```